nixos/modules/services/networking/rosenpass.nix

```diff
@@ -0,0 +1,220 @@
+{ config
+, lib
+, options
+, pkgs
+, ...
+}:
+with builtins;
+with lib; let
+  cfg = config.services.rosenpass;
+  opt = options.services.rosenpass;
+  settingsFormat = pkgs.formats.toml { };
+in
+{
+  options.services.rosenpass = with types; {
+    enable = mkEnableOption "Rosenpass";
+
+    package = mkPackageOption pkgs "rosenpass" { };
+
+    defaultDevice = mkOption {
+      type = nullOr str;
+      description = "Name of the network interface to use for all peers by default.";
+      example = "wg0";
+    };
+
+    settings = mkOption {
+      type = lib.types.submodule {
+        freeformType = settingsFormat.type;
+
+        options = {
+          public_key = mkOption {
+            type = path;
+            description = "Path to a file containing the public key of the local Rosenpass peer. Generate this by running {command}`rosenpass gen-keys`.";
+          };
+
+          secret_key = mkOption {
+            type = path;
+            description = "Path to a file containing the secret key of the local Rosenpass peer. Generate this by running {command}`rosenpass gen-keys`.";
+          };
```

Comment:
here and in many other places lib.mdDoc are missing

Reply:
I thought that `lib.mdDoc` is just the identity function and no longer used: https://github.com/NixOS/nixpkgs/blob/8ab2b1221df4ffaafe779f7c2e4880977695d8d5/lib/options.nix#L349

Please explain why it should be added.

Reply:
Done.